openpower/configs/hostboot/zaius.config

```diff
@@ -68,3 +68,5 @@ set CONSOLE_OUTPUT_FFDCDISPLAY
 unset SECUREBOOT
 unset TPMDD
 unset TPM_NUVOTON
+unset DRTM
```

Comment:
I don't see this config flag in the Hostboot code to make the comment, but why isn't this config:

    `default y if (SECUREBOOT && TPMDD)`
and/or
    `depends on SECUREBOOT && TPMDD`